packages/world/package.json

```diff
@@ -63,7 +63,6 @@
     "ejs": "^3.1.8",
     "forge-std": "https://github.com/foundry-rs/forge-std.git#74cfb77e308dd188d2f58864aaf44963ae6b88b1",
     "glob": "^8.0.3",
-    "hardhat": "^2.10.2",
```

Comment:
iirc this was needed for solhint?

if we don't actually need this then we should prob get rid of a lot more stuff (tasks dir etc)

Reply:
it seems like `solhint` does the same with and without hardhat, interestingly it doesn't find actual errors (like missing semicolon) in either case